docs/content/preview/deploy/multi-dc/async-replication/includes/automatic-setup.md

```diff
@@ -0,0 +1,166 @@
+        --enable \
+        --retention <retention_period> \
+        --database <database_name>
+    ```
+
+    The `retention_period` must be greater than the amount of time you expect the Primary universe to be down before it self recovers or before you perform a failover to the Standby universe.
+
+1. Set up the xCluster replication.
+
+    ```sh
+    ./bin/yugabyted xcluster set_up \
+        --target_address <ip_of_any_target_cluster_node> \
+        --replication_id <replication_id> \
+        --bootstrap_done
+    ```
+
+    You should see output similar to the following:
+
+    ```output
+    +-----------------------------------------------+
+    |                   yugabyted                   |
+    +-----------------------------------------------+
+    | Status        : xCluster set-up successful.   |
+    +-----------------------------------------------+
+
+    ```
+  </div>
+
+  <div id="local" class="tab-pane fade " role="tabpanel" aria-labelledby="local-tab">
+
+<!-- Local Setup -->
+
+The following assumes you have set up Primary and Standby universes. Refer to [Set up universes](../async-deployment/#set-up-universes).
+
+1. Create a checkpoint using the `create_xcluster_checkpoint` command, providing a name for the replication group, and the names of the databases to replicate as a comma-separated list.
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses <primary_master_addresses> \
+        create_xcluster_checkpoint \
+        <replication_group_id> \
+        <comma_separated_namespace_names> \
+        automatic_ddl_mode
+    ```
+
+    The command informs you if any data needs to be copied to the Standby, or only the schema (empty tables and indexes) needs to be created. For example:
+
+    ```output
+    Waiting for checkpointing of database(s) to complete
+    Checkpointing of yugabyte completed. Bootstrap is not required for setting up xCluster replication
+    Successfully checkpointed databases for xCluster replication group repl_group1
+    Create equivalent YSQL objects (schemas, tables, indexes, ...) for databases [yugabyte] on the standby universe
+    Once the above step(s) complete run 'setup_xcluster_replication'
+    ```
+
+    You can also manually check the status as follows:
+
+    ```sh
+    ./bin/yb-admin \
+    -master_addresses 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+    is_xcluster_bootstrap_required repl_group1 yugabyte
+    ```
+
+    You should see output similar to the following:
+
+    ```output
+    Waiting for checkpointing of database(s) to complete
+    Checkpointing of yugabyte completed. Bootstrap is not required for setting up xCluster replication
+    ```
+
+1. If needed, perform a full copy of the database on the Primary to the Standby using distributed backup and restore. See [Distributed snapshots for YSQL](../../../../manage/backup-restore/snapshot-ysql/). Otherwise, create the necessary schema objects (tables and indexes) on the Standby.
```

Comment:
Let's also add a comment here to ensure that all global objects are also created ahead of time